pkg/kube/inject/webhook.go

```diff
@@ -377,7 +378,7 @@ func NewValuesConfig(v string) (ValuesConfig, error) {
 
 type InjectionParameters struct {
 	pod                 *corev1.Pod
-	deployMeta          metav1.ObjectMeta
+	deployMeta          types.NamespacedName
```

Comment:
can we rename it to deployNamespacedName, in case misuse in injector later

Reply:
I don't mind renaming this one, but the struct we actually pass to injection would be nice to keep the same name for compatibility IMO?

Reply:
Yes, i read that